_nonmvptools/capetownbudgetproject.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: tool
-title: Cape Town Budget Project
+name: Cape Town Budget Project
```

Comment:
This is dangerous because it's not clear if it'll overwrite the Jekyll-provided name, unless you're choosing to do that explicitly? It'll mean that people must refer to a project by this long name, rather than the short "capetownbudgetproject" name. Is that what you're aiming for?